### PR TITLE
Add unit tests and lint to Playwright runner

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+  LOG_DIR + '/firefox_browser.log',
+  LOG_DIR + '/supervisord.log',
+  LOG_DIR + '/video-rec-stderr.log',
+  LOG_DIR + '/video-rec-stdout.log',
+  LOG_DIR + '/wait-xvfb.1.log',
+  LOG_DIR + '/wait-xvfb.2.log',
+  LOG_DIR + '/wait-xvfb-stdout.log',
+  LOG_DIR + '/xvfb-tryouts-stderr.log',
+  LOG_DIR + '/xvfb-tryouts-stdout.log',
+  exports.HOME_DIR + '/videos/video.mp4',
```
Comment (vrunoa) [inline src/constants.js:17]: @dpgraham I know is not part of this PR, but can we quickly remove these files since they don't provide any info. Everything above video.mp4 can be removed.

**Review by vrunoa — APPROVED** [review]

**Review by tianfeng92 — APPROVED** [review]

**Review by FriggaHel — APPROVED** [review]
